printer/downloadFile.js

```diff
@@ -0,0 +1,28 @@
+const axios = require('axios');
+const fs = require('fs');
+
+/**
+ * Download PDF file from URL.
+ * @param {Sring} url : PDF file url
+ * @returns filePath: the path to downloaded PDF file
+ */
+async function downloadFileFromURL(url) {
```

Comment:
shouldnt this be returning a promise? we call this function then await either the path to the file or false